keyboards/arabica37/rev1/rev1.c

```diff
@@ -0,0 +1,27 @@
+#include "arabica37.h"
+
+
+#ifdef AUDIO_ENABLE
+    float tone_startup[][2] = SONG(STARTUP_SOUND);
+    float tone_goodbye[][2] = SONG(GOODBYE_SOUND);
+#endif
+
+
+
+void matrix_init_kb(void) {
+
+    #ifdef AUDIO_ENABLE
+        _delay_ms(20); // gets rid of tick
+        PLAY_SONG(tone_startup);
+    #endif
+
+	matrix_init_user();
+};
+
+void shutdown_kb(void) {
+    #ifdef AUDIO_ENABLE
+        PLAY_SONG(tone_goodbye);
+      	_delay_ms(150);
+      	stop_all_notes();
+    #endif
+}
```

Comment:
This entire block should be removed.  this is in quantum, so, you're basically overriding the behavior in a way that can't be easily fixed. 

```suggestion
```